src/auth-service/bin/www

```diff
@@ -30,16 +41,34 @@ try {
 }
 
 
-
 /****
  * pass the app instance to the middleware configuration
  */
 try {
     middlewareConfig(app);
 } catch (e) {
-    console.log("middleware error", e.message);
+    logElement("middleware error", e.message)
 }
 
+
+const consumer = kafka.consumer({ groupId: constants.UNIQUE_CONSUMER_GROUP });
+
+// Function to start consuming messages
+const run = async() => {
+    await consumer.connect();
+    await consumer.subscribe({ topic: constants.NEW_MOBILE_APP_USER_TOPIC, fromBeginning: true });
+    await consumer.run({
+        eachMessage: async({ topic, partition, message }) => {
+            logger.info(
```

Comment:
We shall jus track (log) the Kafka messages (`NEW_MOBILE_APP_USER_TOPIC`) from here. @NoahNsimbe 
If we are able to log them successfully, then we shall be able to route them to our messaging system accordingly.